airbyte-server/build.gradle

```diff
@@ -1,5 +1,26 @@
 plugins {
     id 'application'
+    id 'maven-publish'
+    id 'com.github.johnrengelman.shadow' version '6.1.0'
+}
+
+shadowJar {
+    zip64 true
+    mergeServiceFiles()
+    exclude 'META-INF/*.RSA'
```

Comment:
what is this exclude doing?

Reply:
It removes signed certificates, which may conflict from some packages.